src/DynamoCore/Configuration/PreferenceSettings.cs

```diff
@@ -304,24 +305,14 @@ public bool DisableTrustWarnings
         }
 
         /// <summary>
-        /// Backing store for TrustedLocations
+        /// Default trusted locations. A fixed list of locations (This field should not be serializable or public)
         /// </summary>
-        private List<string> trustedLocations { get; set; } = new List<string>();
+        private List<string> defaultTrustedLocations { get; set; } = new List<string>();
```

Comment:
I'm confused, what is this for? Is it only used during testing?

Reply:
`defaultTrustedLocations` was supposed to be the default trusted paths list (like the programData/appdata dynamo packages folders)
But now that I look at some of the jira [comments](https://jira.autodesk.com/browse/DYN-4905)  I am not so sure that this is required. I will clarify at our next meeting

Reply:
I removed the default trusted locations, as per our discussion